docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md

```diff
@@ -86,19 +85,25 @@ For more details, see the Kubernetes documentation about [Creating a new namespa
    helm repo update
    ```
 
-   > **NOTE**: The Helm chart at https://grafana.github.io/helm-charts is a publication of the source code at grafana/tempo.
+   {{% admonition type="note" %}} The Helm chart at https://grafana.github.io/helm-charts is a publication of the source code at grafana/tempo.
+   {{% /admonition %}}
 
 ## Set Helm chart values
 
-Your Helm chart values are set in the `custom.yaml` file. The following example `custom.yaml` file sets the storage and traces options, enables the gateway, and sets the cluster to main. The `traces` configure the distributor's receiver protocols.
+Your Helm chart values are set in the `custom.yaml` file.
+The Tempo Helm char values example `custom.yaml` file sets the storage and traces options, enables the gateway, and sets the cluster to main.
+The `traces` configure the distributor's receiver protocols.
 
-Next, you will:
+To customize your Helm chart values:
 
-1. Create a `custom.yaml` file
-2. Set your storage values, the example above points to the MinIO instance configured by the chart
-3. Set your traces values to configure the receivers on the Tempo distributor
+1. Create a `custom.yaml` file.
+1. Copy and paste either the Tempo Helm chart values or the GET Helm chart values into your file.
+1. Save the file in your working directory.
+1. Set your storage values, the example helm chart values points to the MinIO instance configured by the chart.
+1. Set your traces values to configure the receivers on the Tempo distributor.
```

Comment:
I'm not sure what to suggest here but I feel we should add some clarification about whether they can just flip the true/false values as needed for their environment, and any other needed clarification, and/or a pointer to a docs page that might add clarification at this point.